### ISSUE TITLE
UX: if you do `--help` it's not clear you're not seeing everything.

### ISSUE THREAD

**Issue by o-jasper** [ideasman42/nerd-dictation#9]
Specifically you don't see `nerd-dictation begin --help`, which basically contains all the goodies?

**Comment by ideasman42** [comment]
Agree, some hint that each sub-command has it's own help would be good to support.
another possibility is expanding all sub commands, although I'm not sure if that might end up being a bit overwhelming.

**Comment by ideasman42** [comment]
I've managed to get this working but would like to see if it can be done more elegantly.

Asked question: https://stackoverflow.com/q/68265036/432509
- Test patch: https://gist.github.com/ideasman42/76ae54bba97cda4163b9e008189c48df
